pkg/apis/keycloak/v1alpha1/keycloakrealm_types.go

```diff
@@ -80,6 +80,34 @@ type KeycloakAPIRealm struct {
 	// user federation at various points.
 	// +optional
 	UserFederationMappers []KeycloakAPIUserFederationMapper `json:"userFederationMappers,omitempty"`
```

Comment:
(A comment unrelated to this specific line.)

Should we add more realm properties like e.g. `clientSessionMaxLifespan`, `accessCodeLifespan`, etc. Should this be 1:1 with the realm representation provided by Keycloak?

Reply:
I have pull requests ready for email, theme, brute force. Just waiting for the rebase.
As for your question, it seems that it should https://issues.redhat.com/browse/KEYCLOAK-14321

Reply:
Thanks! I didn't know there's a JIRA for this already. ;)